src/label/example/index.ts

```diff
@@ -10,6 +10,7 @@ export class App extends WidgetBase<WidgetProperties> {
 			v('h1', {}, ['Label Examples']),
 			v('h3', {}, ['Label assigned as string without extra options']),
 			w(Label, {
+				extraClasses: { root: 'label1' },
```

Comment:
i don't think we should ever show `extraClasses` in an example if we can get away with it as it's kind of a back door / anti pattern. In this case better to wrap in a `div` / `span` with styles